core/src/components/input-password-toggle/input-password-toggle.tsx

```diff
@@ -105,14 +106,51 @@ export class InputPasswordToggle implements ComponentInterface {
     inputElRef.type = inputElRef.type === 'text' ? 'password' : 'text';
   };
 
+  /**
+   * Get the icon to use for the show icon.
+   * If an icon is set on the component, use that.
+   * Otherwise, use the icon set in the config.
+   * If no icon is set in the config, use the default icon.
+   *
+   * @internal
+   * @returns {string} The icon to use for the show icon.
+   */
+  get inputPasswordShowIcon(): string {
```

Comment:
~ Similar to above, I don't think these need to be separate getters. They could be 1 line const.